logify.js

```diff
@@ -0,0 +1,51 @@
+const isPlainObject = require('lodash/isPlainObject');
+const { logger: _logger } = require('./index');
+
+function formatValue(arg) {
+  if (typeof arg === 'function') {
+    return `ƒ() ${arg.name} {...}`;
```

Comment:
fancy

Reply:
it it kosher for us to use unicode in source code now ?

Reply:
i mean it's not python ;)

Reply:
seriously tho, i do it in JS b/c babel compiles escape sequences to the character anyway, so there isn't anything to be gained there unless the editor can't handle it i guess